libraries/tools/kotlin-symbol-processing-api/src/org/jetbrains/kotlin/ksp/symbol/Visibility.kt

```diff
@@ -13,5 +13,6 @@ enum class Visibility {
     PRIVATE,
     PROTECTED,
     INTERNAL,
-    LOCAL
+    LOCAL,
+    JAVA_PACKAGE,
```

Comment:
Since this is an enum, they are better to be orthogonal. Should `JAVA_PACKAGE` be simulated by PUBLIC + package names of accessor and target?

Reply:
Can you elaborate on this? Which other enum is JAVA_PACKAGE overlapping with?

Reply:
PUBLIC